sklearn/manifold/t_sne.py

```diff
@@ -161,7 +161,7 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     dist = pdist(X_embedded, "sqeuclidean")
-    dist /= degrees_of_freedom
+    dist /= float(degrees_of_freedom)
```

Comment:
I would let it as is ensuring that we have `from __future__ import division` on the top